<launch>
    <arg name="model_name" default="albatross"/>
    <arg name="config_pkg" default="$(eval find(arg('model_name') + '_description'))"/>

    <group ns="$(arg model_name)/control_modes">
        <node name="joystick_mode" pkg="tauv_common" type="joystick_mode" output="screen">
            <param name="odom_frame" value="world"/>
            <param name="base_link_frame" value="$(arg model_name)/base_link"/>
            <param name="stab_frame" value="$(arg model_name)/base_link_stabilized"/>
            <param name="output_topic" value="/$(arg model_name)/controllers/cascaded_controller/input"/>
        </node>
    </group>

</launch>